pkg/kubectl/cmd/cmd_test.go

```diff
@@ -201,11 +201,17 @@ func NewAPIFactory() (*cmdutil.Factory, *testFactory, runtime.Codec) {
 		Validator: validation.NullSchema{},
 	}
 	generators := map[string]kubectl.Generator{
```

Comment:
> It is a super-set. It adds "service/test" here: https://github.com/kubernetes/kubernetes/pull/16028/files#diff-bd5a484957da582c1a72fa512a18058eR214

It's a copy/paste instead of "copy this starting point and add", so its hard to keep up to date.  I didn't even know this existed.